### ISSUE TITLE
[all] copyedit for to simplify sentence structure

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-people-use-web-videos#278]
I know this is vague and will be hard to get done, yet I think important that the scripts are edited to simplify sentence structure.

**Comment by nitedog** [comment]
Agree. However, don't forget that most of these videos will be read by a narrator with good voice intonation, flow, and pauses.
